lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -412,6 +412,17 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "Logit") {
```

Comment:
Consider adding a unit test to `caffe2ImporterTest.cpp` to make sure that this code parses the operator out of a Caffe2 protobuf correctly.

Reply:
Done! (Output shape, high level checks on the content of the graph).